### ISSUE TITLE
ImageSegmentationCustomAdvanced problems

### ISSUE THREAD

**Issue by freers623** [Nourepide/ComfyUI-Allor#18]
A strange problems, can choose the model in the ImageSegmentation, but can't choose the model in the ImageSegmentationCustomAdvanced。Please take a look， Thank you！
![allor1](https://github.com/Nourepide/ComfyUI-Allor/assets/11420525/9601fdf1-8ef3-47a2-90a9-801c1f9eff9e)

**Comment by Nourepide** [comment]
Thanks for Issue!

First of all, I'm very surprised that someone actually decided to use ImageSegmentationCustomAdvenced. 🙂

And secondly, have you used ImageSegmentation yet?

The point is that this node downloads models as they are used from the Internet to the onnx folder.

ImageSegmentationCustom and ImageSegmentationCustomAdvanced, unlike their main node, expect the model to be in the onnx folder just before use.

In other words, if you have never used ImageSegmentation, there will be no models in this folder.

**Comment by Nourepide** [comment]
Have you tried restarting ComfyUI?

As far as I know nodes in ComfyUI get the list of files only at the moment of startup.

After ComfyUI is launched, the nodes cannot update the information in any way.

**Comment by freers623** [comment]
When I opened it again today, the node prompt failed, I re-installed the node, the problem is solved! Thank you very much for your work!

**Comment by freers623** [comment]
When I opened it again today, the node prompt failed, I re-installed the node, the problem is solved! Thank you very much for your work!
